Release/fun 4.20

rmoch opened the pull request.

rmoch committed:
✏️Fix typo in certificate template. Closes #4070

rmoch committed:
🚸Organize more conveniently Django admin Course list view. Closes #4076

Remove useless column and add usefull ones
Reduce column header by css
Reduce label text
Reduce text input width

jmaupetit reviewed (approved):
Happy to see that CSS what the easiest solution :wink:

rmoch committed:
✏️(forum) fix typo in french i18n Closes #4073

rmoch committed:
🎨(certificate) improve PDF certificate generation. Closes #4068

Use 3 lines, instead of 2, for small caracters mention to allow correct display
of universities whit long names.
